### ISSUE TITLE
Do tfjs-posenet model and tflite-posenet model have the same weights?

### ISSUE THREAD

**Issue by darcula1993** [tensorflow/tfjs#2403]
I am working on build an android app based on tflite-posenet model which comes from `https://github.com/tensorflow/examples/tree/master/lite/examples/posenet/android`.

However, after I compared the performance of tflite model with the performance of tfjs posenet demo (both of them use mobilenet v1_100 as backend), I found the tfjs demo is much more accurate.

Do they use same weights?

**Comment by darcula1993** [comment]
I am also working on building keras version posenet from previous posenet weights `https://storage.googleapis.com/tfjs-models/weights/posenet/mobilenet_v1_100/manifest.json`. The model behavior is also different from tfjs-posenet demo behavior.

Are they using different weights?

**Comment by dsmilkov** [comment]
cc @tylerzhu-github (the creator of PoseNet) for my context.

**Comment by emredog** [comment]
I'm also curious about the answer. Thanks in advance.

**Comment by tylerzhu-github** [comment]
Hi @darcula1993, @emredog,

Thx for your inquery! Currently both the TFLite PoseNet models and the latest TF.js  PoseNet models are converted from the same checkpoints (with batch normal variables folded into the preceding convolution layer). The difference been TF.js models supports different output stride e.g. 8, 16, 32 and configurable input resolutions, which can affect the accuracy. Let me know if this answers your questions and look forward to hearing more about your progress on building Keras PoseNet.  @dsmilkov feel free to also include the main contacts of the TFLite Android PoseNet app and iOS PoseNet app, they will know more about the details. 

Best,
Tyler

**Comment by darcula1993** [comment]
> Hi @darcula1993, @emredog,
> 
> Thx for your inquery! Currently both the TFLite PoseNet models and the latest TF.js PoseNet models are converted from the same checkpoints (with batch norm variables folded into the preceding convolution layer). The differences that could cause the regression is that TF.js models supports different output stride e.g. 8, 16, 32 and configurable input resolutions, which can affect the accuracy. Let me know if this answers your questions and look forward to hearing more about your progress on building Keras PoseNet. @dsmilkov feel free to also include the main contacts of the TFLite Android PoseNet app and iOS PoseNet app, they will know more about the details.
> 
> Best,
> Tyler

thank you very much for your reply! It did help!

**Comment by rthadur** [comment]
thank you closing this issue , feel free to reopen if you need more info.
